apps/bfd-server/bfd-server-war/src/test/java/gov/cms/bfd/server/war/r4/providers/CarrierClaimTransformerV2Test.java

```diff
@@ -81,7 +81,8 @@ public void transformSampleARecord() throws FHIRException {
 
     assertMatches(
         claim,
-        CarrierClaimTransformerV2.transform(new MetricRegistry(), claim, Optional.of(false)));
+        CarrierClaimTransformerV2.transform(new MetricRegistry(), claim, Optional.of(false)),
```

Comment:
What is this doing? CarrierClaimTransformerV2.transform(new MetricRegistry(), claim, Optional.of(false)),        false);